modules/swagger-codegen/src/main/resources/Java/StringUtil.mustache

```diff
@@ -48,4 +48,12 @@ public class StringUtil {
     if (o == null) return "null";
     return o.toString().replace("\n", "\n    ");
   }
+
+  public static boolean isNotBlank(String s){
```

Comment:
This is the Apache Commons implementation:
```java
    public static boolean isBlank(final CharSequence cs) {
        int strLen;
        if (cs == null || (strLen = cs.length()) == 0) {
            return true;
        }
        for (int i = 0; i < strLen; i++) {
            if (Character.isWhitespace(cs.charAt(i)) == false) {
                return false;
            }
        }
        return true;
    }
```

I think the edge case this captures over yours is "non-blank" whitespace like tabs, returns, etc...granted, this is currently specific to whatever `MDC.get()` returns (does it return `null` or empty string if it's not found?) so it may not need to handle as much.